client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,161 @@
+		this.props.goToStep( 2 );
+	},
+
+	goToStepThree() {
+		this.props.goToStep( 3 );
+	},
+
+	goToStepFour() {
+		this.props.goToStep( 4 );
+	},
+
+	skipStep() {
+		this.props.goToNextStep();
+	},
+
+	renderStepContent() {
+		const connectionToJetpackComplete = true;
+
+		const checkMark = <Gridicon
+			className="plan-features__item-checkmark"
+			size={ 18 } icon="checkmark" />;
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header">Steps you've completed:</div>
+								<ul className="jpo-summary__completed-onboarding">
+									{ 'undefined' !== typeof this.props.signupProgress[0].jpoSiteTitle ? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[1].jpoSiteType ? <li>{ checkMark } { translate( 'Type of Site' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[2].jpoHomepage ? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[3].jpoContactForm ? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li> : null }
+									{ connectionToJetpackComplete ? <li>{ checkMark } { translate( 'Connection to Jetpack' ) }</li> : null }
+								</ul>
+							</td>
+							<td>
+								<div className="jpo__summary-col-header">Configure more of your site:</div>
+								<ul className="jpo-summary__more-onboarding">
+									{ 'undefined' === typeof this.props.signupProgress[0].jpoSiteTitle ? <li onClick={ this.goToStepOne }>{ translate( 'Site Title & Description' ) }</li> : null }
+									{ 'undefined' === typeof this.props.signupProgress[1].jpoSiteType ? <li onClick={ this.goToStepTwo }>{ translate( 'Type of Site' ) }</li> : null }
+									{ 'undefined' === typeof this.props.signupProgress[2].jpoHomepage ? <li onClick={ this.goToStepThree }>{ translate( 'Type of Homepage' ) }</li> : null }
+									{ 'undefined' === typeof this.props.signupProgress[3].jpoContactForm ? <li onClick={ this.goToStepFour }>{ translate( 'Contact Us Form' ) }</li> : null }
+									{ ! connectionToJetpackComplete ? <li>{ translate( 'Connection to Jetpack' ) }</li> : null }
+									<li>{ translate( 'Choose a Theme' ) }</li>
+									<li>{ translate( 'Add a Site Address' ) }</li>
+									<li>{ translate( 'Add a Store' ) }</li>
+									<li>{ translate( 'Start a Blog' ) }</li>
+								</ul>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+				<div>
+					<Button primary onClick={ this.completeOnboarding }>Complete Onboarding</Button>
+				</div>
+			</div>
+		);
+	},
+
+	render() {
+		let siteTitle = ( 'undefined' !== typeof this.props.signupProgress[0].jpoSiteTitle )
+			? this.props.signupProgress[0].jpoSiteTitle.siteTitle
+			: translate( 'your new site' );
+
+		const headerText = translate( 'Congratulations! ' ) + siteTitle + translate( ' is on its way.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **70** times:
`translate( 'Congratulations!' )` *ES Score: 33*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).